### PR TITLE
Kiwix.KiwixJS.Electron version 1.7.2-E

### PR DESCRIPTION
**Pull request by Jaifroid**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

File: manifests/k/Kiwix/KiwixJS/Electron/1.7.2_E/Kiwix.KiwixJS.Electron.locale.en-GB.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kiwix.KiwixJS.Electron
+PackageVersion: 1.7.2_E
```
Comment (ItzLevvie) [inline manifests/k/Kiwix/KiwixJS/Electron/1.7.2_E/Kiwix.KiwixJS.Electron.locale.en-GB.yaml:4]:
```suggestion
PackageVersion: 1.7.2-E
```

File: manifests/k/Kiwix/KiwixJS/Electron/1.7.2_E/Kiwix.KiwixJS.Electron.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kiwix.KiwixJS.Electron
+PackageVersion: 1.7.2_E
```
Comment (ItzLevvie) [inline manifests/k/Kiwix/KiwixJS/Electron/1.7.2_E/Kiwix.KiwixJS.Electron.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 1.7.2-E
```

File: manifests/k/Kiwix/KiwixJS/Electron/1.7.2_E/Kiwix.KiwixJS.Electron.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kiwix.KiwixJS.Electron
+PackageVersion: 1.7.2_E
```
Comment (ItzLevvie) [inline manifests/k/Kiwix/KiwixJS/Electron/1.7.2_E/Kiwix.KiwixJS.Electron.yaml:4]:
```suggestion
PackageVersion: 1.7.2-E
```

File: manifests/k/Kiwix/KiwixJS/Electron/1.7.2_E/Kiwix.KiwixJS.Electron.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kiwix.KiwixJS.Electron
+PackageVersion: 1.7.2_E
```
Comment (ItzLevvie) [inline manifests/k/Kiwix/KiwixJS/Electron/1.7.2_E/Kiwix.KiwixJS.Electron.installer.yaml:4]:
```suggestion
PackageVersion: 1.7.2-E
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please make sure to update the version folder to be `1.7.2-E` instead of `1.7.2_E`.

**Comment by Jaifroid** [comment]
@ItzLevvie Thanks for noticing the inconsistency with 1.7.2_E vs 1.7.2-E. Darn. So I think possibly I need to close and re-do this PR because it will end up in the wrong directory? 1.7.2_E is in the manifest path on winget...  I guess this is why validation has just failed.

**Pull request closed without merging**